pkg/auth/manager.go

```diff
@@ -89,6 +89,13 @@ func (a *AuthManager) handleAuthRequest(_ context.Context, key signalAuthKey) er
 		authType:       policy.AuthType(key.AuthType),
 	}
 
+	if k.localIdentity.IsReservedIdentity() || k.remoteIdentity.IsReservedIdentity() {
```

Comment:
i wonder whether it would be better to perform this check in the datapath. It's kind of unnecessary to trigger user-space via signalmap

Reply:
This would be possible but we then need to also extend thinking futher here as we should send some signal to somehow be able to point the user that there might be a misconfiguration causing this...

Reply:
fair point. as discussed offline when I raised this point - i'm totally fine by merging as is.